doc/gov/chapters/chapter05.md

```diff
@@ -33,8 +33,19 @@ Being on the voting representative list **doesn't** mean the right to vote. The
 
 <sup>2)</sup>The criteria will be checked and the voting representative list will be updated prior any voting event when the voting process kicks in.
 
-<sup>3)</sup> Individuals who don't present an organisation will not have voting right.
+<sup>3)</sup> Individuals who don't represent an organisation will not have voting right.
 
 <a name="5.3"></a>
 ## 5.3 Voting Process
-
+ 
+ In Scenarios where Consensus cannot be reached, a voting event will be triggered by Technical/Governance Steering Leads (depending on if the event corresponds to a technical or governance matter). 
+
+ When a voting event is triggered, an online voting mechanism will be setup and the following process will be followed:
+ 
+ - The time and the deadline of which the voting representatives are required to cast their vote will be determined by the leads themselves.
+ - The question will be created by the leads (A simple clear two-choices question is expected (**A | B**)).
+ - The vote result will be announced by the leads after the voting deadline.
+ - More than **50%** of the valid votes are needed for a decision to be made (Between **A | B**).
```

Comment:
Should the approval vote percent be higher? If it is just over 50% then there is really a divide. Secondly, what if the majority of eligible voting members do not cast a vote?
Also, should we qualify so that we have a mix of operators and vendors in the approval mix.

Suggest changes -- the numbers are arbitrary and may be tweaked:
1. At least 80% of the eligible voting representatives have cast a vote
2. At least two-thirds of valid votes cast are needed for a decision to be made  (between **A | B**) and at least one-third of the votes in favour of the decision should be from the operators or vendors. Explanatory Note: say, 75% votes were cast in favour of decision A, then at least 25% of the votes should be from operators or vendors.

Reply:
I'm a little hesitant to require too many strings on a decision; it's a recipe for getting nothing done.  (That said, if the goal is to promote stability by making it more difficult to be impulsive, go for it.)

Reply:
I agree with @nickchase I won't put too many restrictions. after all, when there is a fundamental difference, we need to resolve it and get to the bottom of it and get consensus. voting is intended when we start to get two options to choose one from. I would suggest keeping it simple and we can adjust if we feel there was a need to, @pgoyal01 @nickchase

Reply:
@nickchase @rabi-abdel can we at least require a certain level of participation so as not to have a decision made by a few -- say, 75% of representatives should have voted.

Reply:
i am not sure about it @pgoyal01 that will make it pretty much impossible to get a decision out.